lambdas/build-cri-oauth-request/src/main/java/uk/gov/di/ipv/core/buildcrioauthrequest/BuildCriOauthRequestHandler.java

```diff
@@ -334,12 +334,18 @@ private EvidenceRequest getEvidenceRequestForF2F(List<SignedJWT> credentials)
                 requiredEvidences.stream()
                         .filter(
                                 requiredScores ->
-                                        requiredScores.getEvidences().size() == 1
-                                                & requiredScores.getActivity() == 0
-                                                & requiredScores.getFraud() == 0
-                                                & requiredScores.getVerification() <= 3)
-                        .map(requiredScores -> requiredScores.getEvidences().get(0))
-                        .mapToInt(Gpg45Scores.Evidence::getStrength)
+                                        requiredScores.getEvidences().size() <= 1
```

Comment:
This makes a lot of sense given our understanding that you can get verification scores from F2F , which means you may not need any evidence requirements to still make a valid request to F2F CRI

Reply:
Yep - I think this _might_ happen if you fail the 'liveness/likeness' check in DCMAW, so have a strength/validation score, but no verification (but I don't fully understand it tbh).